controller/action/AcquisitionActions.class.php

```diff
@@ -27,14 +27,24 @@ public static function executeYouTubeSub()
 
         return Controller::redirect(Request::getReferrer(), 303);
     }
-
-    public static function executeYouTube()
+      
+    public static function executeYouTube(string $version = '')
     {
         if (isset($_GET['error_message'])) {
             $error_message = Request::encodeStringFromUser($_GET['error_message']);
         }
 
-        return ['acquisition/youtube', [
+        $baseTemplate = 'acquisition/youtube';
+        $versionedTemplate = $baseTemplate . '-' . $version;
+        $template = $version && View::exists($versionedTemplate) ? $versionedTemplate : $baseTemplate;
+
+        if ($version && View::exists($versionedTemplate)) {
+            Session::set("template", $template);
```

Comment:
"template" needs to be a const and probably shouldn't be named so simply (session values are global).